### ISSUE TITLE
Training a model using this dataset

### ISSUE THREAD

**Issue by tushar-nitave** [google-research-datasets/query-wellformedness#1]
I want to train a model using this dataset so that when I feed it with _grammatically incorrect_ questions it should rate it. How can create such model?

**Comment by MartinLichtblau** [comment]
Well, that's the thing you have to find out. And now that you have the labeled data you can try out which NN-architecure performs best on that task. The hope is that it finds out by itself which one is syntactically wrong. I would even consider looking for grammar checkers, like in MS Word or grammarly; perphas like this you come up with some creative ideas to improve SOTA deep learning approaches. And you might wanna use sth. like [Universal Sentence Encoder](https://arxiv.org/pdf/1803.11175.pdf) to create some kind of embeddings. 
Just search the web and combine what you find — be creative :-) 

Edit: Here is what you want (the associated paper) [Identifying Well-formed Natural Language Questions](http://aclweb.org/anthology/D18-1091).

**Comment by tushar-nitave** [comment]
Thank you for suggestion.

**Comment by tushar-nitave** [comment]
@MartinLichtblau I tried training this using neural networks with different hyperparameter setting but, I am not able to achieve accuracy above 30%. I used Universal Sentence Encoder for embedding the dataset. Can you provide some help?

**Comment by MarcusSorealheis** [comment]
@tushar-nitave Are you still experiencing trouble?

**Comment by ola-elkhatib** [comment]
Is there any pretrained model that evaluates arbitrary questions?
